packages/renderer/src/lib/ui/NavPage.svelte

```diff
@@ -9,7 +9,7 @@ export let searchEnabled: boolean = true;
   <div class="min-w-full">
     <div class="flex">
       <div class="pt-5 px-5">
-        <p class="text-xl capitalize">{title}</p>
+        <p class="text-xl">{title}</p>
```

Comment:
Let's use css to apply uppercase on the first letter (and then no need to add uppercase at other locations)

```suggestion
        <p class="text-xl first-letter:uppercase">{title}</p>
```